Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/common/certificate/DccMaxPersonChecker.kt

```diff
@@ -35,7 +35,7 @@ class DccMaxPersonChecker @Inject constructor(
         if (allIdentifiers.size == allIdentifiersWithNew.size) return Result.Passed
 
         // adding the certificate results in exceeding max -> block import
-        if (allIdentifiersWithNew.size > max) {
+        if (allIdentifiersWithNew.size >= max) {
             Timber.i("Maximum exceeded. Max is $max, no of persons is ${allIdentifiersWithNew.size}")
```

Comment:
this is not correct - it says it should show up when exceeded

Reply:
See https://github.com/corona-warn-app/cwa-app-tech-spec/pull/145/files#diff-25c8b7188514d7cdfb430f7f8f963fed63bca279fd74753bc6cefe29dd599b55R1982